qa/tasks/scripts/ses_replace_disk_part1.sh

```diff
@@ -0,0 +1,28 @@
+set -ex
+
+declare -a random_minion_fqdn="$@"
+
+echo "### Getting random minion and its random OSD ###"
+random_minion=`echo $random_minion_fqdn | cut -d . -f 1`
+random_osd=`ceph osd tree | grep -A 1 $random_minion | grep -o "osd\.".* | awk '{print$1}'`
+osd_id=`echo $random_osd | cut -d . -f 2`
```

Comment:
If OSD has id like `some.name.ID`, it is better to use bash instead of calling extra bash instance with cut:
```
osd_id=${random_osd##*.}
```